harness/tests/determined/common/experimental/test_determined.py

```diff
@@ -126,7 +126,7 @@ def test_default_retry_doesnt_retry_allowed_status(
 
 
 @responses.activate
-def test_get_trial_populates_summary_metrics(make_client: Callable[[], Determined]) -> None:
+def test_get_trial_populates_all_parameters(make_client: Callable[[], Determined]) -> None:
```

Comment:
If this is parameterizable with `pytest.mark.parameterize`, then could you please use that? If not, then [the docs](https://docs.pytest.org/en/7.3.x/how-to/parametrize.html#basic-pytest-generate-tests-example) talk about how to generate more complex parameterizations.

I don't expect it to be difficult, one way or the other. If it is, then let's chat and maybe we can figure something else out.